articles/quickstart/native/flutter/interactive.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Add login to your Flutter app
+default: true
+description: This tutorial demonstrates how to add user login with Auth0 to a Flutter application using the Auth0 Flutter SDK
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - flutter
+  - dart
+  - ios
+  - android
+contentType: tutorial
+useCase: quickstart
+interactive: true
+files:
+  - files/build
+  - files/main
+---
+
+# Add login to your Flutter app
+
+Auth0 allows you to quickly add authentication and access user profile information in your application. This guide demonstrates how to integrate Auth0 with a Flutter application using the [Auth0 Flutter SDK](https://github.com/auth0/auth0-flutter).
+
+## Getting started
+
+This quickstart assumes you already have a [Flutter](https://flutter.dev/) application up and running. If not, check out the [Flutter "getting started" guides](https://docs.flutter.dev/get-started/install) to get started with a simple app, or clone [our sample apps](https://github.com/auth0-samples/auth0-flutter-samples).
+
+You should be familiar with the [Flutter command line tool](https://docs.flutter.dev/reference/flutter-cli).
+
+<%= include('_configure_urls_interactive') %>
+
+## Install the Auth0 Flutter SDK
+
+Add the Auth0 Flutter SDK into the project:
+
+```shell
+flutter pub get auth0-flutter
+```
+
+## Configure Android {{{ data-action=code data-code="build.gradle#11" }}}
+
+If you are not developing for the Android platform, you can skip this step.
+
+The SDK requires manifest placeholders. Auth0 uses placeholders internally to define an `intent-filter`, which captures the authentication callback URL. You must set the Auth0 tenant domain and the callback URL scheme.
+
+You do not need to declare a specific `intent-filter` for your activity, because you have defined the manifest placeholders with your Auth0 **Domain** and **Scheme** values and the library will handle the redirection for you.
+
+In the snippet, we are using values referenced from `strings.xml`:
+
+- `com_auth0_domain`: The domain of your Auth0 tenant. Generally, you can find this in the Auth0 Dashboard under your Application's Settings in the Domain field. If you are using a custom domain, you should set this to the value of your custom domain instead.
+- `com_auth0_scheme`: The scheme to use. Can be a custom scheme, or `https` if you want to use [Android App Links](https://auth0.com/docs/applications/enable-android-app-links). You can read more about setting this value in the [Auth0.Android SDK README](https://github.com/auth0/Auth0.Android#a-note-about-app-deep-linking).
+
+Run **Sync Project with Gradle Files** inside Android Studio to apply your changes.
+
```

Comment:
This is missing a "Configure iOS" section.

Reply:
Added in 0c4275800. This just borrowed from the Swift quickstart.